chromedriver.log

```diff
@@ -0,0 +1,4 @@
+Starting ChromeDriver 86.0.4240.22 (398b0743353ff36fb1b82468f63a3a93b4e2e89e-refs/branch-heads/4240@{#378}) on port 9515
+Only local connections are allowed.
+Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
+ChromeDriver was started successfully.
```

Comment:
Probably best not to commit the log files. Consider adding:
```
/chromedrive.log
/geckodrive.log
```
to the `.gitignore` file to prevent them from being commited.